core/new-gui/src/app/workspace/component/code-editor-dialog/code-editor-dialog.component.ts

```diff
@@ -36,20 +37,21 @@ export class CodeEditorDialogComponent implements AfterViewInit, SafeStyle, OnDe
     fontSize: "11",
     automaticLayout: true,
   };
-  code: string;
   @ViewChild("editor", { static: true }) divEditor: ElementRef | undefined;
   loaded: boolean = false;
 
+  private formControl: FormControl;
   private ytext?: YText;
```

Comment:
I know this is unrelated for this PR, but since you removed the code variable now, I believe this variable is the actual code for the editor. can we name it as "code" with type "YText"?

Reply:
Done.